ansible_collections/arista/avd/roles/eos_designs/python_modules/base/avdstructuredconfig.py

```diff
@@ -40,6 +47,22 @@ def _mgmt_ip(self):
         """
         return get(self._hostvars, "switch.mgmt_ip")
 
+    @cached_property
+    def _ipv6_mgmt_ip(self):
+        """
+        Returns the value for switch.ipv6_mgmt_ip fact used in management_interfaces data-models
+        """
+        return get(self._hostvars, "switch.ipv6_mgmt_ip")
+
+    @cached_property
+    def _mgmt_ipv6_enable(self):
```

Comment:
this is redundant. You can just test the `self._ipv6_mgmt_ip is not None` where you need it.

Reply:
Fixed, thanks!